src/game/AI/ScriptDevAI/scripts/outland/hellfire_citadel/hellfire_ramparts/boss_watchkeeper_gargolmar.cpp

```diff
@@ -136,6 +136,15 @@ struct boss_watchkeeper_gargolmarAI : public CombatAI
                 if (m_creature->GetHealthPercent() < 40.0f)
                 {
                     DoScriptText(SAY_HEAL, m_creature);
+
+                    CreatureList WatcherList;
+                    GetCreatureListWithEntryInGrid(WatcherList, m_creature, NPC_HELLFIRE_WATCHER, 100.0f);
+                    for (auto& itr : WatcherList)
```

Comment:
Itr implies its an iterator. This is very much a creature, so if possible, name it that way. `Creature* watcher :` is waaay better to read.

Reply:
I mostly copy pasted it by another script, should be solved now